doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -247,7 +243,7 @@ described elsewhere
 
       .. coqtop:: none reset
 
-         Require Import PeanoNat.
+         Notation "x 'mod' y" := (Nat.modulo x y) (at level 40) : nat_scope.
```

Comment:
There is nothing to display in the next `Search` commands.

Reply:
I wanted to use `exists` but apparently the thing only works for `mod`, or only for infix operators I don't know.